sentry/src/main/java/io/sentry/SentryStartupProfilingOptions.java

```diff
@@ -0,0 +1,146 @@
+package io.sentry;
+
+import io.sentry.vendor.gson.stream.JsonToken;
+import java.io.IOException;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+final class SentryStartupProfilingOptions implements JsonUnknown, JsonSerializable {
+
+  boolean profileSampled;
```

Comment:
should these be final?

Reply:
nope, as these are read and deserialized from json (L100)